composer.json

```
@@ -76,14 +79,16 @@
             "@fix:php:cs"
```

```suggestion
            "@fix:php:rector"
            "@fix:php:cs"
```

Rector before cs because rector, for example, does not add `use` alphabetically which cs afterwards cleans up.

---

At least locally this does not work

I have to run fix without ddev and fix:php:rector with ddev

---

Okay, will check later.

---

The other checks depend on `Build/Scripts/runTests.sh` and they need docker, you can therefore not run them in ddev.

rector runs directly and therefore needs the PHP Version etc of composer. Therefore you need to run it in ddev depending on your local setup.

---

Now all checks are running in docker.